bjoern.py

```diff
@@ -70,6 +86,8 @@ def run(*args, **kwargs):
     """
     global _default_instance
 
+    kwargs, statsd_config = __separate_config(kwargs)
```

Comment:
Why not make the statsd config param as follows

```
def run(*args, **kwargs):
    statsd = kwargs.pop('statsd', None)
```

```
run(statsd={
    'host': ...,
    'port': ....,
})
```

If `statsd` not passed -> disabled.

Type checks in the `.run()` method.

`statsd_` params in `server_run` remain as-is but with `None` defaults.